README.md

```diff
@@ -23,9 +23,10 @@ We use JIRA instead of the issues page on Github. If you would like to see what
 ## Java client library
 for sbt:
 
-```libraryDependencies += "org.broadinstitute.dsde.workbench" %% "leonardo-client" % "0.1-<git hash>"```
+```libraryDependencies += "org.broadinstitute.dsde.workbench" %% "leonardo-client" % "1.3.6-<git hash>"```
```

Comment:
what exactly is the logic behind this versioning?

Reply:
Not related to this PR per say, just some general clean up of our java client library doc. Not sure what the actual logic behind versioning is really, which is why this is an example and the user should look at what is available on Jfrog (link provided below)